docs/site/Model.md

```diff
@@ -65,6 +65,11 @@ decorators for adding metadata to your TypeScript/JavaScript classes in order to
 use them with the legacy implementation of the
 [datasource juggler](https://github.com/strongloop/loopback-datasource-juggler).
 
+## Model Discovery
+
+Some connectors support the ability to
+[Discover](Discovering-models-from-relational-databases.md) models.
```

Comment:
Let's move this section _after_ "Definition of a Model" please.

Here is how I am thinking about the layout of this page:

- Overview
- Definition of a Model
- Model Discovery
- Using the Juggler Bridge

I am also proposing to slightly rephrase the text.

```markdown
LoopBack can automatically create model definitions by discovering the schema 
of your database. See 
[Discovering models](Discovering-models-from-relational-databases.md)
for more details and a list of connectors supporting model discovery.
```

Reply:
done